CS483/CS483/Kartaclysm/Components/Abilities/ComponentBedazzleAbility.cpp

```diff
@@ -107,7 +107,7 @@ namespace Kartaclysm
 		{
 			if (m_pConditions->CanActivate() &&
 				pAIDriver->DistanceToClosestOpponent() <= 2.0f &&
-				rand() % 8 == 0)
+				rand() % 30 == 0)
```

Comment:
I think it's fine, it's mostly just to make them less likely to nail you the moment you pass them.
Of course, we can revisit if necessary.